app/controllers/surveys_controller.rb

```diff
@@ -13,6 +13,7 @@ def send_survey
       return render json: {error: :survey_not_found}, status: :not_found
     end
 
+    @survey.generate_models
```

Comment:
Can we have a check if the models are already generated don't generate them again? (And a test)

For the case where the user might want to send out more surveys for respondents they forgot. At this point the survey is immutable.